diff --git a/pytype/typeshed b/pytype/typeshed
deleted file mode 120000
index e4d38331..00000000
--- a/pytype/typeshed
+++ /dev/null
@@ -1 +0,0 @@
-../typeshed
\ No newline at end of file
diff --git a/setup.py b/setup.py
index e535703f..d5ccf8cb 100644
--- a/setup.py
+++ b/setup.py
@@ -110,8 +110,6 @@ def get_version():
 
 def get_install_requires():
   requires = [
-      'importlab (>=0.5.1)',
-      'ninja',
       'pyyaml (>=3.11)',
       'six',
   ]
